docs/data-sources/modelarts_datasets.md

```diff
@@ -0,0 +1,114 @@
+---
+subcategory: "Beta"
+---
+
+# huaweicloud_modelarts_datasets
+
+Use this data source to get a list of ModelArts datasets.
+
+## Example Usage
+
+```hcl
+data "huaweicloud_modelarts_datasets" "test" {
+  name = "your_dataset_name"
+  type = 1
+}
+
+output "dataset_id" {
+  value = data.huaweicloud_modelarts_datasets.test.datasets[0].id
+}
+```
+
+## Argument Reference
+
+The arguments of this data source act as filters for querying the available datasets in the current region.
+ All datasets that meet the filter criteria will be exported as attributes.
+
+* `region` - (Optional, String) Specifies the region in which to obtain datasets. If omitted, the provider-level region
+ will be used.
+
+* `name` - (Optional, String) Specifies the name of datasets.
+
+* `type` - (Optional, String) Specifies the type of datasets. The options are:
+  + **0**: Image classification, supported formats: `.jpg`, `.png`, `.jpeg`, `.bmp`.
+  + **1**: Object detection, supported formats: `.jpg`, `.png`, `.jpeg`, `.bmp`.
+  + **3**: Image segmentation, supported formats: `.jpg`, `.png`, `.jpeg`, `.bmp`.
+  + **100**: Text classification, supported formats: `.txt`, `.csv`.
+  + **200**: Sound classification, Supported formats: `.wav`.
+  + **400**: Table type, supported formats: Carbon type.
+  + **600**: Video, supported formats: `.mp4`
+  + **900**: Free format.
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `id` - Indicates a data source ID.
+
+* `datasets` - Indicates a list of all datasets found. Structure is documented below.
+
+The `datasets` block contains:
+
+* `id` - The ID of the datasets.
+
+* `name` - The name of the datasets.
+
+* `type` - The type of the datasets.
+
+* `description` - The description of the datasets.
+
+* `output_path` - The OBS path for storing output files such as labeled files.
+
+* `data_source` - The data sources which be used to imported the source data (such as pictures/files/audio, etc.) in
+ this directory and subdirectories to the dataset. Structure is documented below.
+
+* `schemas` - The schema information of source data when `type` is `400`(Table Type). Structure is documented below.
+
+* `labels` - The labels information. Structure is documented below.
+
+* `data_format` - The dataset format. Valid values include: `Default`, `CarbonData`: Carbon format(Supported only for
+ table type datasets).
+
+* `created_at` - The dataset creation time.
+
+* `status` - Dataset status. Valid values are as follows:
+  + *0*: Creating.
+  + *1*: Completed.
+  + *2*: Deleting.
+  + *3*: Deleted.
+  + *4*: Exception.
+  + *5*: Syncing.
+  + *6*: Releasing.
+  + *7*: Version switching.
+  + *8*: Importing.
```

Comment:
please use bold instead of italics to keep the fomat consistent.

Reply:
ok